plugin-server/src/worker/ingestion/event-pipeline/populateTeamDataStep.ts

```diff
@@ -29,35 +29,23 @@ export async function populateTeamDataStep(
         })
         .inc()
 
-    // If a team_id is present (event captured from an app), trust it and return the event as is.
-    if (event.team_id) {
-        // Check for an invalid UUID, which should be blocked by capture, when team_id is present
-        if (!UUID.validateString(event.uuid, false)) {
-            await captureIngestionWarning(db.kafkaProducer, event.team_id, 'skipping_event_invalid_uuid', {
-                eventUuid: JSON.stringify(event.uuid),
-            })
-            throw new Error(`Not a valid UUID: "${event.uuid}"`)
-        }
-
-        return event as PluginEvent
-    }
```

Comment:
Where did this block go?

Reply:
It's identical to the block on line 59, except we no longer early-return because we need to fetch the team to do the person processing flag check